index.js

```diff
@@ -1,5 +1,16 @@
+/* eslint-disable no-unused-vars */
 const humburgerBtn = document.querySelector('.humburger');
 const navMenu = document.querySelector('.nav-menue');
+const Modal = document.querySelector('.modal-container');
+const ModalTitle = document.querySelector('.modaltitle');
+const ModalTechnology = document.querySelector('.language_used_modal');
+const ModalImage = document.querySelector('.popup-image');
+const ModalDescription = document.querySelector('.pop-up-description');
+const ModalSeeLive = document.querySelector('.seelive');
+const ModalSeeSource = document.querySelector('.seesource');
+const Company = document.querySelector('.pop-company');
+const Role = document.querySelector('.pop-role');
+const Year = document.querySelector('.pop-year');
```

Comment:
- [ ] I think it would be better to name your variables using lower camelCase instead of PascalCase, the last is often used by convention for defining classes 🤖.  This will help you follow widely accepted conventions as well as keep consistency throughout your entire codebase. Please, consider fixing this🤓 🛠️ 

[Read more here](https://developer.mozilla.org/en-US/docs/MDN/Writing_guidelines/Writing_style_guide/Code_style_guide/JavaScript#variables).